src/main/java/com/snack/news/repository/SlackBotRepository.java

```diff
@@ -3,5 +3,11 @@
 import com.snack.news.domain.slack.SlackChannel;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.Iterator;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused import - java.util.Iterator.](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5523973)